### PR TITLE
Auto add issue to TODOs when opened

### PR DESCRIPTION
**Pull request by Arcticae**

### CODE REVIEW

File: .github/workflows/assign_issues_to_project.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to Project
+    steps:
+    - name: Assign new issues to the project board
+      uses: srggrs/assign-one-project-github-action@1.2.1
+      if: github.event.action == 'opened'
+      with:
+        project: 'https://github.com/orgs/software-mansion/projects/2'
+        column_name: 'TODO'
```
Comment (MaksymilianDemitraszek) [inline .github/workflows/assign_issues_to_project.yml:19]: I wonder if this shouldn't be backlog.

**Review by MaksymilianDemitraszek — APPROVED** [review]

File: .github/workflows/assign_issues_to_project.yml (hunk 1 above)
Comment (MaksymilianDemitraszek) [reply .github/workflows/assign_issues_to_project.yml:19]: I mean not every issue we open is TODO, maybe it should move to TODO if somebody has been assigned to it?

Comment (Arcticae) [reply .github/workflows/assign_issues_to_project.yml:19]: IMHO i think we should at least look at it before we throw it into the abyss of backlog :P

Comment (MaksymilianDemitraszek) [reply .github/workflows/assign_issues_to_project.yml:19]: maybe that's a good point

**Pull request merged**